package-lock.json

```diff
@@ -14,34 +14,45 @@
       }
     },
     "@babel/core": {
-      "version": "7.4.3",
-      "resolved": "https://registry.npmjs.org/@babel/core/-/core-7.4.3.tgz",
-      "integrity": "sha512-oDpASqKFlbspQfzAE7yaeTmdljSH2ADIvBlb0RwbStltTuWa0+7CCI1fYVINNv9saHPa1W7oaKeuNuKj+RQCvA==",
+      "version": "7.4.5",
```

Comment:
Please checkout a fresh copy of package-lock.json.

Reply:
Hi @apb7, 
Sorry missed this, also this `package-lock.json` seems to be fresh as or running `start.sh` there was no update on  `package-lock.json`